app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
```

Comment:
This is not the most preferrable way to create objects that are associated to each other. Because a job has many comments, we can do...

```ruby
job = Job.find(params[:job_id])
@comment = job.comments.new(comment_params)
```

this way, we're leveraging our associations 👍